src/bindings/js.ts

```diff
@@ -1094,28 +1094,28 @@ export class JSBuilder extends ExportsWalker {
       sb.push(name);
       if (clazz.extends(this.program.staticArrayPrototype)) {
         // optional last argument for __lowerStaticArray
-        let valueType = clazz.getArrayValueType();
-        if (valueType.isNumericValue) {
+        let type = clazz.getArrayValueType();
```

Comment:
Name `type` here shadows an outer variable of the same name. Perhaps better to keep it as `valueType`?

Reply:
Ah, right!